beacon/src/beacon.c

```diff
@@ -354,45 +416,25 @@ static void _beacon_epoch_()
     }
 }
 
-int _set_adv_data_()
-{
-#ifdef BEACON_PLATFORM__GECKO
-    log_debug("Setting legacy adv data...\r\n");
-    print_bytes(payload.en_data.bytes, MAX_BROADCAST_SIZE, "adv_data");
-    sl_status_t sc = sl_bt_advertiser_set_data(legacy_set_handle,
-                                               0, 31,
-                                               payload.en_data.bytes);
-    if (sc != 0)
-    {
-        log_errorf("Error, sc: 0x%lx\r\n", sc);
-        return -1;
-    }
-    log_debug("Success!\r\n");
-#endif
-    return 0;
-}
-
 static int _beacon_advertise_()
 {
     int err = 0;
 #ifdef BEACON_PLATFORM__ZEPHYR
-    // Start advertising
     err = bt_le_adv_start(
         BT_LE_ADV_PARAM(
-            BT_LE_ADV_OPT_USE_IDENTITY, // use random identity address
-            BEACON_ADV_MIN_INTERVAL,
-            BEACON_ADV_MAX_INTERVAL,
-            NULL // undirected advertising
-            ),
+            BT_LE_ADV_OPT_USE_IDENTITY,
+            BEACON_ADV_MIN_INTERVAL, BEACON_ADV_MAX_INTERVAL,
+            NULL),
```

Comment:
can we restore the comments? I included them purposefully

Reply:
Aah yep, will do!